.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "http://json.schemastore.org/prettierrc",
+  "printWidth": 80,
```

Comment:
remove unnecessary config

Reply:
I think `trailingComma: "all"` will be nicer

Reply:
good idea.